src/main/java/teammates/common/util/Const.java

```diff
@@ -1357,6 +1370,14 @@ private Const() {
         public static final String FEEDBACK_SESSION_PUBLISHED =
                 "The feedback session has been published. "
                 + "Please allow up to 1 hour for all the notification emails to be sent out.";
+        public static final String FEEDBACK_SESSION_RESEND_LINKS_NOT_PUBLISHED =
+                "The feedback session is not published. "
+                + "You cannot resend published notification emails for a session that is not published.";
+        public static final String FEEDBACK_SESSION_RESEND_LINKS_EMPTY_RECIPIENT =
+                "You have not selected any student to email.";
+        public static final String FEEDBACK_SESSION_RESEND_LINKS_EMAIL_SENT =
```

Comment:
On second thoughts, change the prefix `FEEDBACK_SESSION_RESEND_LINKS_` -> `FEEDBACK_SESSION_RESEND_EMAIL_`. This constant can be `FEEDBACK_SESSION_RESEND_EMAIL_SENT`.